### ISSUE TITLE
Leave button when viewing own profile is misleading

### ISSUE THREAD

**Issue by NotAFile** [vector-im/riot-android#2138]
When you click your own avatar in a room, you are led to this screen:

![image](https://user-images.githubusercontent.com/5447747/38444819-21478110-39f1-11e8-8620-3a276db4c045.png)

As you can see, there is an option to leave the room that is marked with a large X.

This is unfortunate, because the X is usually associated with "close this". This means that distracted users might intuitively click it, wanting to close the fragment, but then accidentally leave the room instead. The standard "open door" leave symbol might be more adequate.

Inspired by a true story:tm:

**Comment by ac4000** [comment]
Agreed, this should be moved elsewhere, does not require the X, and should probably have a confirmation dialog
